playwright/pages/ordersPage.ts

```diff
@@ -126,8 +125,6 @@ export class OrdersPage extends BasePage {
     console.log("Navigating to order details view: " + orderLink);
     await this.page.goto(orderLink);
     await this.waitForDOMToFullyLoad();
-    await this.waitForGrid();
-    await this.waitForGrid();
```

Comment:
question: why was this causing errors? why did we had to wait previously for grid and now it works fine 🤔